src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+
+export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationSettingsToIgnore: string[], destinationName: string): Promise<void> {
+    const addedKeys: string[] = [];
+    const updatedKeys: string[] = [];
+    const userIgnoredKeys: string[] = [];
+    const matchingKeys: string[] = [];
+
+    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
+    const options: vscode.QuickPickItem[] = [];
+    destinationSettingsToIgnore.length = 0;
+
+    for (const element of Object.keys(sourceSettings)) {
+        options.push({
+            label: element
+        });
+    }
+
+    const result = await context.ui.showQuickPick(options, {
+        placeHolder: 'Select the app settings you would like to download:',
+        isPickSelected: (item) => { return !listOfSettingsToIgnore.includes(item.label) }
+        , canPickMany: true
```

Comment:
I thought this was fixed in a previous PR, but can you put this on the previous line

Reply:
Fixed.